test/unit/foreman/renderer/renderers_shared_tests.rb

```diff
@@ -39,6 +39,11 @@ module RenderersSharedTests
       assert_equal 'http://foreman.some.host.fqdn', renderer.render(source, @scope)
     end
 
+    test "plugin_present? helper method" do
+      source = OpenStruct.new(content: '<%= plugin_present?("crazy_plugin") %>')
+      assert_equal 'false', renderer.render(source, @scope)
+    end
+
```

Comment:
this should do for positive test
```
test "plugin_present? finds existing plugin" do
  Foreman::Plugin.register('my_plugin') {}
  source = OpenStruct.new(content: '<%= plugin_present?("my_plugin") %>')
  assert_equal 'true', renderer.render(source, @scope)
end
```

Reply:
Thanks, i'll try that.

Reply:
It turned out, `Foreman::Plugin.register(:my_plugin) {}` is the way to go. Uff